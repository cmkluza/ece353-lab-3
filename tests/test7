Tests LW and SW, with no errors, with m = n = 1, c = 2
./sim-mips -b 1 1 2 ./tests/asm/test3.s ./tests/out/test7.out
./tests/sols/test3.sol