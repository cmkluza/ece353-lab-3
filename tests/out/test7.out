program name: ./tests/asm/test3.s
stage utilization: 0.866667  0.400000  0.400000  0.400000  0.300000 
register values 0  0  0  0  0  0  0  1  40  250  2591  256  32767  0  0  256  32767  2591  0  0  0  0  0  0  0  0  0  0  0  0  0  52
